_publications/2017_jpf_dr.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: publication
+# The quotes make the : possible, otherwise you can do it without quotes
+title: "Consistency-Aware Scheduling for Weakly Consistent Programs"
+# YYYY_CONFSHORTNAME_PAPERSHORTNAME
+key: 2017_jpf_dr
+
+# conference | journal | poster | workshop
+type: workshop
+redirect_from: /publications/2017_jpf_dr
+
+# The shortname is used for auto-generated titles
+shortname: JPF 2017
+# add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
+image: 2017_jpf_dr.png
+# add a 2:1 aspect ratio teaser figure (e.g., width: 1200px, height: 600px) to the folder /assets/images/papers/
+image_large: 2017_jpf_dr_teaser.png
+
+# Authors in the "database" can be used with just the key (lastname). Others can be written properly.
+authors:
+- dabaghchian
+- rakamaric
+
+journal-short: JPF
+year: 2017
+
+# bibentry: article | inproceedings | phdthesis | book
+bibentry: article
+bib:
```

Comment:
You are missing a bunch of bibtex fields.
```
@article{jpf2017-dromt,
  author = {Maryam Dabaghchian and Zvonimir Rakamari\'c and
    Burcu K. Ozkan and Erdal Mutlu and Serdar Tasiran},
  title = {Consistency-Aware Scheduling for Weakly Consistent Programs},
  journal = {ACM SIGSOFT Software Engineering Notes},
  volume = {42},
  number = {4},
  year = {2017},
  month = {October},
  pages = {1--5},
  doi = {10.1145/3149485.3149493},
  publisher = {ACM},
  note = {Proceedings of the 2017 Java Pathfinder Workshop (JPF)},
}
```

Reply:
Done.